### DISCUSSION TITLE
Mensagem de falta de suporte ao .Net 5

### DISCUSSION THREAD

**Discussion by Fehard22** [balta-io/forum#126] [Tira Dúvidas] [answered]
Olá, Balta!
Primeiramente gostaria de agradecer pelo conteúdo disponibilizado gratuitamente. Não sei se terei como pegar o Premium dessa vez, infelizmente, mas assim que possível irei adquirir, com certeza!
Minha dúvida é a seguinte. Estou trabalhando com o .Net 5 e sempre que dou run, aparece a mensagem de que não há mais suporte e blablabla. Tem como ocultar essa mensagem? Não interfere em nada, só me incomoda mesmo, visualmente falando. 
Desde já, agradeço.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Fehard22, como vai?
Promos incríveis da black friday rolando => https://balta.io/premium

Sobre a mensagem, infelizmente acredito que não tem como remover ela, é um warning da própria SDK :(
Talvez seja hora de migrar para o .NET 6 ou 7 hahahahah

**Reply by Fehard22** [reply]
Eu inicialmente até instalei a versão 7, mas a estrutura do código estava bem diferente, então fiquei com o 5. Vou mantê-lo por enquanto. Muito obrigado pela atenção, Balta!
